composer.json

```
@@ -16,6 +16,7 @@
         "laravel/pint": "^1.13",
         "laravel/sail": "^1.41",
         "mockery/mockery": "^1.6",
+        "nunomaduro/collision": "^8.1",
```

```suggestion
        "nunomaduro/collision": "^8.6",
```

Only Collision 8.6 and above can be used on Laravel 12.

---

It was remove since it's not possible to be used with Laravel 12. Check `conflict` section in Collision's composer.json

---

Gotcha. Thanks. Guess I'll close then.

---

Actually your PR is valid, just 8.1-8.5 will be ignored to conflict. You can use 8.6 and above

---

Oh, I understand now. I'll reopen.